src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media_Animation/Given_DoubleAnimation.cs

```diff
@@ -116,37 +120,42 @@ async Task WaitAndSnapshotValue(int millisecondsDelay)
 
 		[TestMethod]
 		[RunsOnUIThread]
-		public async void When_RepeatForever_ShouldLoop_AsdAsd()
+		public async Task When_RepeatForever_ShouldLoop()
```

Comment:
What the existing test failing ? If if did not, let's create another one.

Reply:
Answered above.